report-api/src/test/java/no/sikt/nva/data/report/api/fetch/testutils/generator/TestData.java

```diff
@@ -211,7 +248,7 @@ private TestNviContributor generateNviContributor() {
 
     private TestNviOrganization generateNviAffiliation() {
         return TestNviOrganization.builder()
-                   .withId(randomUri().toString())
+                   .withId(organizationUri("10.1.1.2"))
```

Comment:
The string "10.1.1.2" is used several places. Maybe make it a constant? A public one if you need to assert on it in tests.